packages/sampler/stories/default/ExpandableInput.js

```diff
@@ -1,12 +1,11 @@
 import ExpandableInput, {ExpandableInputBase} from '@enact/moonstone/ExpandableInput';
-import {icons} from '@enact/moonstone/Icon';
+import iconNames from './icons';
```

Comment:
Should this same change be applied in other files that import `icons` from `@enact/moonstone/Icon`?

Reply:
It might be clearer for me to say: Should we re-use the icons file from the default sampler in the qa sampler?

Reply:
"Can't resolve './icons' in '/Users/lucie.roy/Work/enact/packages/sampler/stories/qa'"
It was one of the reason I left it as it was ... although would have like the code to be similar across. Any suggestion please?

Reply:
tried to adjust the path but it is not fixing the error either.

Reply:
I think Roy meant it should refer to the `icons.js` file in the default folder, so: `'../default/icons'`

Reply:
It would probably require moving the `icons.js` file into `../../utils/` and then including it from there.

Reply:
In theory, you could import from `../default/icons.js` but that seems lazy.

Reply:
Personally, I don't feel it's "correct" to move Moonstone's icon list into the general util directory. Meanwhile, it's more appropriate to have Moonstone's QA folder refer to stuff in Moonstone's "default" folder.

Reply:
Well...  OK.

Reply:
Thank you to both... the path fixed it. I was using the incorrect path and the incorrect file but it was the idea (Blake fixed the path). It works for at least one. Will test the others.

Reply:
If we're always sorting it, we could export the sorted list from `icons.js`.

Reply:
That seems like an ok approach too.

```
const sortedIcons = Object.keys(icons).sort();

export default Object.keys(icons);
export {decrementIcons, incrementIcons, listIcons, mediaIcons, arrowIcons, starIcons, sortedIcons};
```

Reply:
I will submit what I have and you can decide for next steps